### PR TITLE
[cli] update coaps readme

### PR DESCRIPTION
**Pull request by jrhodie**
Please review updates to the coaps start command.

### CODE REVIEW

File: src/cli/README_COAPS.md [hunk 1]
```diff
@@ -236,11 +236,26 @@ Sets the content sent by the test resource.
 Done
 ```
 
-### start
+### coaps start \[check-peer-cert\|max-conn-attempts\]
```
Comment (Vyrastas) [inline src/cli/README_COAPS.md:239]: Should "coaps" be here?  None of the other commands listed use it

Comment (abtink) [reply src/cli/README_COAPS.md:239]: +1. 

I agree. we should not need the `coaps` here

**Review by abtink — APPROVED** [review]
LGTM. Thanks.

Just same comment above.

File: src/cli/README_COAPS.md (hunk 1 above)
Comment (jrhodie) [reply src/cli/README_COAPS.md:239]: thanks, will fix now

**Comment by jrhodie** [comment]
@Vyrastas @abtink - made the requested change, thanks.

**Review by Vyrastas — APPROVED** [review]

**Review by jwhui — APPROVED** [review]

**Pull request merged**